frontend/testing/playwright/pages/DataModelPage.ts

```diff
@@ -66,7 +66,7 @@ export class DataModelPage extends BasePage {
   }
 
   public async clickOnAddNodeToPropertyButton(): Promise<void> {
-    await this.getButtonByTextKey('schema_editor.add_node_of_type').click();
+    await this.getButtonByTextKey('schema_editor.add_node_of_type').last().click();
```

Comment:
Why would you like the last element? 😄

Reply:
In this code, both the button on the top and the buttons in the tree view have the same name, so we need to tell Playwright how to find the right one. Ideally, we should use something similar to `within` in Testing Library and Cypress, but reading up on the Playwright documentation I couldn't find any such functionality except from CSS selectors, which we want to avoid since they are more unstable. However, we know that the button we need is always the last one of the two, so therefore I think it's reasonable to use `last()` here. But it's not ideal indeed.